### DISCUSSION TITLE
V1 release timeline

### DISCUSSION THREAD

**Discussion by roguh** [nats-io/nats.py#212] [Q&A] [answered]
When will v1 be released? I'd like to update a project to Python 3.10.

**Comment by wallyqs** [comment, marked as answer]
I'm really hoping that some time next week it is ready! It would actually be a v2 bump to better signal that it there are breaking changes in the API.

**Reply by henloef** [reply]
Am I correct in assuming v2 is what will be the result of #210?

**Comment by roguh** [comment]
That's excellent! Thank you for the quick reply.

**Comment by roguh** [comment]
Any updates?

**Reply by wallyqs** [reply]
I will release soon after this merges... https://github.com/nats-io/nats.py/pull/216
